pages/client_login.php

```diff
@@ -60,20 +37,96 @@
             </div>
         </div>
         <div class="sign_up_menu">
-            <input class="input" type="text" name="username_signup" placeholder="Username" id="username" onblur="checkUsername($('#username').val())"><br>
-            <input class="input" type="text" name="email_signup" placeholder="Email"><br>
-            <input class="input" type="password" name="password_signup" placeholder="Password"><br>
-            <input class="input" type="password" name="password_signup_confirm" placeholder="Confirm Password"><br>
-            <button class="user_login" name="user_login">
+            <input class="input_signup" type="text" name="username_signup" placeholder="Username" id="username" onblur="checkUsername()"><span class="warning">Username Taken</span><br>
+            <input class="input_signup" type="text" name="email_signup" placeholder="Email"><br>
+            <input id='password'class="input_signup" type="text" name="password_signup" placeholder="Password" onblur="checkPass()"><br>
```

Comment:
it should be password, i just changed it to text for testing purposes and forgot to change it back, updating now